### ISSUE TITLE
sanitizer-common: Linux/get_sock_peer_name.cpp fails without outgoing network iface

### ISSUE THREAD

**Issue by mgorny** [llvm/llvm-project#53721]
The listed test fails with the following assertion:
```
get_sock_peer_name.cpp.tmp: /var/tmp/portage/sys-libs/compiler-rt-sanitizers-14.0.0.9999/work/compiler-rt/test/sanitizer_common/TestCases/Linux/get_sock_peer_name.cpp:27: int main(): Assertion `connect(fd, reinterpret_cast<const sockaddr *>(&sin), sizeof(sin)) == 0' failed.
```

After debugging it a bit, it turns out it's trying to connect to 1.0.0.127 which isn't available in our isolated network namespace. I've submitted a patch to fix endianness in https://reviews.llvm.org/D119461.

**Comment by tamird** [comment]
No idea! @vitalybuka may know.

**Comment by tamird** [comment]
It probably is fine.

**Comment by tamird** [comment]
It would be nice to enable the test on android if you think it passes now.

**Comment by tstellar** [comment]
Merged: 1772de7212e9
